common/omv_csi.c

```diff
@@ -260,12 +260,8 @@ __weak int omv_csi_abort(omv_csi_t *csi, bool fifo_flush, bool in_irq) {
     csi->last_frame_ms = 0;
     csi->last_frame_ms_valid = false;
 
-    if (csi->fb) {
-        if (fifo_flush) {
-            framebuffer_flush_buffers(csi->fb, true);
-        } else if (!csi->disable_full_flush) {
-            framebuffer_flush_buffers(csi->fb, false);
-        }
```

Comment:
What was going on here previously is:

fifo_flush==True

Flush the frame buffer and don't leave any valid frames to get via the get_fb() call.

fifo_flush==False && !csi->disable_full_flush

Flush the frame buffer, but, leave the current frame valid to get via the get_fb() call.

...

Given a way to grab the current frame was removed from the csi API, is the intention here that any get_fb() like functionality is to be removed?

Even so, csi->disable_full_flush is still valuable to prevent the frame buffer fifo from being emptied on abort. Suggest:

```
if (csi->fb && fifo_flush && (!csi->disable_full_flush)) {
    framebuffer_flush(csi->fb);
}
```

Reply:
> I think acquire(HEAD|PEEK) is fine to grab the frame, if this returns NULL then you should return None for the FB image object right? I think that makes sense.

Calling `acquire(FREE|PEEK)` from non-snapshot code, should never fail especially if it follows a `resize(1)`. I don't know/remember every use case, you should test these corner cases and let me know if something needs to be fixed.

Reply:
Okay, will do.

Reply:
Not seeing any issues. Things are working as expected.